WebRoot/messages/ZMsg_iw.properties

```diff
@@ -39,18 +39,18 @@ account.PASSWORD_LOCKED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e0\u05e2\u05d
 account.PASSWORD_RECENTLY_USED = \u05d4\u05e1\u05d9\u05e1\u05de\u05d4 \u05e9\u05e9\u05dc\u05d7\u05ea \u05d4\u05d9\u05d9\u05ea\u05d4 \u05d1\u05e9\u05d9\u05de\u05d5\u05e9 \u05dc\u05d0\u05d7\u05e8\u05d5\u05e0\u05d4. \u05d1\u05d7\u05e8 \u05e1\u05d9\u05e1\u05de\u05d4 \u05d0\u05d7\u05e8\u05ea.
 
 # {0} = filename
-formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 '{0}'.
+formatter.INVALID_FORMAT = \u05ea\u05d1\u05e0\u05d9\u05ea \u05e7\u05d5\u05d1\u05e5 \u05dc\u05d0-\u05d7\u05d5\u05e7\u05d9\u05ea \u05e2\u05d1\u05d5\u05e8 \u05d4\u05e7\u05d5\u05d1\u05e5 ''{0}''.

```

Comment:
@mihaipopahtp Standardization is certainly a good thing. 
Just to clarify my previous point: the lines where I fixed the representation of apostrophes are just a portion of the total: merely the ones I could identify through the presence of format constructs.
Any other string used as a format which isn't one of those is still broken when it comes to apostrophes.
So the patch is a partial fix.
There are two or three options that I can see to fix the problem in its entirety:
(I) check string by string in the code to see whether they are literal strings or formats and write the apostrophes accordingly. **A lot** of work which would not prevent issues with strings added in the future or future modifications to existing string values. Also hard to check for errors in an automatic fashion.
(II) Get rid of all apostrophes everywhere, replacing them with a single character. The smart quote we both mentioned (U+2019) would be a decent candidate, because, even if it doesn't fit perfectly as I mentioned in the previous comment, it could be used where an apostrophe would be spot on and also as an enclosing character where one might use a double quote for instance.
(III) Get rid of all apostrophes using a human translator for each language to decide which character to use to replace them with _at every point_ in the strings. Obviously the best option, but also the most time-consuming and the one involving the most people.
Both (II) and (III) have the advantage of eliminating the problem at the root, also allowing to spot future problems just by checking for the presence of apostrophes in the text. Among the two, (II) which you suggested is most likely the best compromise. It's not perfect but should prevent glaring failures in the localized output from occurring again and it amounts to little more than a search and replace. IMHO an incorrect use of the quote would be preferable to a broken localized string.
My two (or four given the length ;-) ) cents.
Of course, any of the three options is an improvement over the current situation.